buffer.go

```diff
@@ -0,0 +1,139 @@
+package kinesis
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"html/template"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kinesis"
+	"github.com/gliderlabs/logspout/router"
+)
+
+// PutRecordsLimit is the maximum number of records allowed for a PutRecords request.
+var PutRecordsLimit = 500
```

Comment:
I feel like maybe we should have this be lower - or maybe let it be configurable if possible?  Just worried about our less noisy services not producing output.

Even better, maybe a time based flush?